src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.TargetFrameworkInference.targets

```diff
@@ -250,4 +250,19 @@ Copyright (c) .NET Foundation. All rights reserved.
     <OutputPath>$(OutputPath)$(TargetFramework.ToLowerInvariant())\</OutputPath>
   </PropertyGroup>
 
+  <UsingTask TaskName="GenerateSupportedTargetFrameworkAlias" AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
+
+  <!--
+    Generate a list of valid TargetFramework aliases for the project system to consume for the target framework properties page drop down
+  -->
+  <Target Name="GenerateSupportedTargetFrameworkAlias">
```

Comment:
I think this should say `Returns="@(SupportedTargetFrameworkAlias)`.

Reply:
I'm using the outputs attribute instead here, I think it's the same thing with different syntax: https://docs.microsoft.com/en-us/visualstudio/msbuild/target-element-msbuild?view=vs-2019#attributes

Reply:
The Outputs is on the _task_ not the target. We'll need the `Returns` or `Outputs` on the target to grab these values.